docs/application/dotnet/getting-started/features.md

```diff
@@ -0,0 +1,47 @@
+<a id="features"></a>
+# Tizen .NET Features and Components
+
+Tizen .NET enables you to build .NET applications with Xamarin.Forms and the Tizen .NET framework. Xamarin.Forms allows you to easily create a user interface, and the TizenFX API provides numerous interfaces to much of the hardware that is found in modern TV, mobile, wearable, and IoT devices.
+
+**Figure: Tizen .NET architecture**
+
+![Tizen .NET architecture](media/cs_overview.png)
+
+Tizen .NET consists of the following main components:
+
+- **.NET Core**
+
+  .NET Core is a general-purpose development platform maintained by Microsoft and the .NET community on [GitHub](https://github.com/dotnet/core). It is cross-platform, supporting Windows&reg;, macOS, and Linux, and can be used in device, cloud, and embedded/IoT scenarios.
+
+  .NET Core is composed of the following parts:
+
+  - The [.NET runtime](https://github.com/dotnet/coreclr), which provides basic services, such as a type system, assembly loading, a garbage collector, and native interop.
+  - A set of [framework libraries](https://github.com/dotnet/corefx), which provide primitive data types, application composition types, and fundamental utilities.
+  - A [set of SDK tools](https://github.com/dotnet/cli) and [language compilers](https://github.com/dotnet/roslyn) that enable the base developer experience, available in the [.NET Core SDK](https://docs.microsoft.com/en-us/dotnet/articles/core/sdk).
+  - The dotnet application host, which is used to launch .NET Core applications. It selects and hosts the runtime, provides an assembly loading policy, and launches the application. The application host is also used to launch SDK tools similarly.
+
+- **Xamarin.Forms as a UI framework**
+
+  Xamarin.Forms is a cross-platform UI toolkit that allows you to efficiently create native user interface layouts that can be shared across iOS, Android&trade;, Windows Phone, and Universal Windows Platform applications.
```

Comment:
Suggestion:
Xamarin.Forms is a cross-platform UI toolkit, which allows you to efficiently create native user interface layouts that can be shared across iOS, Android&trade;, Windows phone, and Universal Windows Platform (UWP) applications.

Reply:
done